test/ArborX_EnableDeviceTypes.hpp.in

```diff
@@ -1,13 +1,10 @@
 #ifndef ARBORX_ENABLE_DEVICE_TYPES_HPP
 #define ARBORX_ENABLE_DEVICE_TYPES_HPP
 
-#include <Kokkos_Macros.hpp>
-
-#include <boost/utility/identity_type.hpp>
+#include <Kokkos_Core.hpp>
```

Comment:
Do we need the Kokkos header at all here?

Reply:
We need the definitions for the device types.

Reply:
I was going to say no you don't but now I realize you traded the macro for an alias.

Reply:
I could try to see if reverting that still works if you want. In the end, we have to include that header anyway, though, and I prefer using as little macros as necessary.